app/components/manage-intermissions/edit.hbs

```diff
@@ -1,3 +1,5 @@
+
+<div {{did-update this.updatedIntermission @intermissionToEdit}}>
```

Comment:
no need to pass in `@intermissionToEdit` here I think

Reply:
tried without it, doesnt work